pkg/synopsysctl/cmd_deploy.go

```diff
@@ -113,7 +114,7 @@ var deployCmd = &cobra.Command{
 		log.Debugf("creating Synopsys-Operator components")
 		soperatorSpec := soperator.NewSOperator(deployNamespace, synopsysOperatorImage, exposeUI, adminPassword, postgresPassword,
 			userPassword, blackduckPassword, secretType, operatorTimeBombInSeconds, dryRun, logLevel, threadiness, postgresRestartInMins,
-			podWaitTimeoutSeconds, resyncIntervalInSeconds, terminationGracePeriodSeconds, sealKey, restconfig, kubeClient, cert, key)
+			podWaitTimeoutSeconds, resyncIntervalInSeconds, terminationGracePeriodSeconds, sealKey, restconfig, kubeClient, cert, key, admissionWebhookListener)
```

Comment:
@jdartigalongue Is update scenario missing?

Reply:
Correct. As stated in the description, we can't rely on the admission webhook and it currently has very little use since it doesn't do much.  Therefore, I don't think that we should add flags to enable/disable to synsopsysctl just yet. Let me know if you don't agree though.